capnp/lib/capnp.pyx

```diff
@@ -2522,12 +2681,13 @@ cdef class TwoPartyClient:
         deref(self._pipe._pipe.ends[1]).write(
             write_buffer.data.as_voidptr,
             len(data)
-        ).wait(self._pipe._event_loop.thisptr.waitScope)
+        ).wait(deref(self._pipe._event_loop.waitScope))
 
     def __dealloc__(self):
         del self.thisptr
 
     cpdef _connect(self, host_string):
+        # TODO: Make async
```

Comment:
Can KJ help with this?

Reply:
See https://github.com/capnproto/pycapnp/issues/311 for my proposal w.r.t. file descriptors. This would solve async connection issue.